pkg/karpenter/karpenter.go

```diff
@@ -12,28 +12,37 @@ import (
 const (
 	// DefaultKarpenterNamespace default namespace for Karpenter
 	DefaultKarpenterNamespace = "karpenter"
+	// DefaultKarpenterServiceAccountName is the name of the service account which is needed for Karpenter
+	DefaultKarpenterServiceAccountName = "karpenter"
 
-	karpenterHelmRepo         = "https://charts.karpenter.sh"
-	karpenterHelmChartName    = "karpenter/karpenter"
-	karpenterReleaseName      = "karpenter"
-	controllerClusterName     = "controller.clusterName"
-	controllerClusterEndpoint = "controller.clusterEndpoint"
-	createServiceAccount      = "serviceAccount.create"
-	addDefaultProvisioner     = "defaultProvisioner.create"
+	karpenterHelmRepo      = "https://charts.karpenter.sh"
+	karpenterHelmChartName = "karpenter/karpenter"
+	karpenterReleaseName   = "karpenter"
```

Comment:
same here - it seems redundant to include `karpenter` in the name of these vars since they are in pkg `karpenter`